.github/workflows/deploy-docs.yml

```diff
@@ -33,6 +33,11 @@ jobs:
           --file requirements.txt
           --file requirements-dev.txt
 
+    # For some reason Fiona doesn't get installed from requirements-dev...
+    - name: Install fiona dependency
+      shell: bash -l {0}
+      run: python -m pip install fiona
```

Comment:
If you can wait I'll take a look at this next week. Something is not right here...

Reply:
something's weird right! I can definitely wait, very interesting to figure out what's going on.

Note that https://github.com/python-visualization/folium/pull/2007 is waiting for the fix to get the docs going again. Waiting a week is not an issue, but if it's much longer then maybe a quick fix is better. Definitely not meant to rush you! More of an expectation thing.

I did also manage to fix the Selenium heatmap test issue in this PR, but I'll spin that out in a separate one and merge that earlier.

Reply:
I split the underlying issues for the Selenium heatmap and this change. Once we have both fixed shall we make a new release?

Reply:
sounds good!

Reply:
> something's weird right! I can definitely wait, very interesting to figure out what's going on.

Found it. See #2012. Basically we are limiting to what is on `main` to build/test the docs.